rfc/rfc-65/rfc-65.md

```diff
@@ -0,0 +1,163 @@
+## Proposers
+
+- @stream2000
+- @hujincalrin
+- @huberylee
+- @YuweiXiao
+
+## Approvers
+
+## Status
+
+JIRA: [HUDI-5823](https://issues.apache.org/jira/browse/HUDI-5823)
+
+## Abstract
+
+In some classic hudi use cases, users partition hudi data by time and are only interested in data from a recent period
+of time. The outdated data is useless and costly, we need a TTL(Time-To-Live) management mechanism to prevent the
+dataset from growing infinitely.
+This proposal introduces Partition TTL Management policies to hudi, people can config the policies by table config
+directly or by call commands. With proper configs set, Hudi can find out which partitions are outdated and delete them.
```

Comment:
Calling commands doesn't work for Flink engine. What's the way of partition ttl management for Flink side.